website/docs/_posts/assets/js/fx.js

```diff
@@ -0,0 +1 @@
+document.title += " | Renovate";
```

Comment:
Website assets shouldn't be stored in this repository. Is there no other way to get the suffix using mkdocs natively?